### PR TITLE
Large tophat!

### PR DESCRIPTION
**Pull request by Bierkraan**
## Why's this needed? 

It's cool hip and new

### CODE REVIEW

File: config/admins.txt [hunk 1]
```diff
@@ -2,4 +2,6 @@
 #The available ranks can be found in /proc/rank_to_level().
 #Most of the time, you'll want to make yourself a Host.
 #To do so, type your BYOND key with no spaces, capitals or other punctuation (including underscores!), a dash with spaces on both side, then the rank you want.
-#For example, the key "ZeWaka" would be given Host rank like this: zewaka - Host
\ No newline at end of file
+#For example, the key "ZeWaka" would be given Host rank like this: zewaka - Host
+
+bierkraan - Host
```
Comment (zjdtmkhzt) [inline config/admins.txt:7]:
```suggestion
```
Probably best not do push this (shelterfrog)

**Review by zjdtmkhzt — CHANGES_REQUESTED** [review]

**Comment by Bierkraan** [comment]
> ```suggestion
```
Probably best not do push this (shelterfrog) 

oops forgot to change that

File: config/admins.txt (hunk 1 above)
Comment (moonlol) [reply config/admins.txt:7]: your suggested change would remove the zewaka line.

**Comment by zjdtmkhzt** [comment]
There is a merge conflict with hand_headgear.dmi.
You might need to update your local version of it to the newest and then readd your changes to the new version or something.

File: code/obj/item/clothing/hats.dm [hunk 2]
```diff
@@ -422,6 +422,13 @@
 		..()
 		src.setMaterial(getMaterial("gold"))
 
+/obj/item/clothing/head/LongTophat
```
Comment (ZeWaka) [inline code/obj/item/clothing/hats.dm:425]: the capitalization is gross

File: config/admins.txt (hunk 1 above)
Comment (ZeWaka) [inline config/admins.txt:7]:
```suggestion
```

**Comment by Kyle2143** [comment]
What do you mean by (semi) tested? Also, what about adding it to the clothing booth? or does that not work with bit hats?

**Comment by Bierkraan** [comment]
> What do you mean by (semi) tested? Also, what about adding it to the clothing booth? or does that not work with bit hats?

at the time of uploading (without errors) it was 10PM and was to tired to do anything really changing it

**Pull request merged**